db/schema.rb

```diff
@@ -25,6 +25,24 @@
     t.index ["user_id"], name: "index_comments_on_user_id"
   end
 
+  create_table "friends", force: :cascade do |t|
+    t.bigint "user_id"
+    t.integer "status"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["user_id"], name: "index_friends_on_user_id"
+  end
```

Comment:
You should use your seeds file to create fake users so that we can try the friendship when you implement the front end. This way is easier since you don't have to create users manually. You can check this [article](https://www.xyzpub.com/en/ruby-on-rails/3.2/seed_rb.html) to start.

Reply:
We have created the Fake Users and implemented Seeds file for the database.